sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client.hpp

```diff
@@ -73,6 +73,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
         std::shared_ptr<Core::Credentials::TokenCredential const> credential,
         KeyClientOptions options = KeyClientOptions());
 
+    /**
+     * @brief Construct a new Key Client object from another key client.
+     *
+     * @param keyClient An existing key vault key client.
+     */
+    explicit KeyClient(KeyClient const& keyClient) : m_pipeline(keyClient.m_pipeline) {}
```

Comment:
Why do we need this new copy ctr?

Reply:
Because the Pageble needs to have a client inside of it, so, we need to be able to init a client from itselft :)

Reply:
Why doesn't the implicit copy-constructor work in this scenario?